modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -186,13 +205,59 @@ struct IEUnit {
     // This method is [supposed to be] called at Island compilation stage
     cv::gimpl::ie::IECompiled compile() const {
         auto this_plugin = IE::PluginDispatcher().getPluginByDevice(params.device_id);
```

Comment:
Isn't `getPluginByDevice()` deprecated by IE? If so, there is a `getPluginByName()` function.

Reply:
Moving to the new "IE Core" API is in the TODO list

Reply:
I thought some fixme would be nice but ok

Reply:
Will put, thanks!

Reply:
Done